### PR TITLE
Lambda Release March 2022 - Update sample layers and existing versions

### PR DESCRIPTION
**Pull request by vasireddy99**
This PR updates the published layer ARNs and the updated versions to links

### CODE REVIEW

File: src/docs/getting-started/lambda/lambda-java.mdx [hunk 1]
```diff
@@ -35,12 +35,17 @@ In this section, we consume the Lambda layer for use with Java Lambda Functions.
...
+| ap-northeast-1<br/>ap-northeast-2<br/>ap-south-1<br/>ap-southeast-1<br/>ap-southeast-2<br/>ca-central-1<br/>eu-central-1<br/>eu-north-1<br/>eu-west-1<br/>eu-wes t-2<br/>eu-west-3<br/>sa-east-1<br/>us-east-1<br/>us-east-2<br/>us-west-1<br/>us-west-2 | arn:aws:lambda:\<region\>:901920570463:layer:aws-otel-java-wrapper-amd64-ver-1-11-1:1 | Contains [OpenTelemetry for Java v1.11.1](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.11.1) with [Java Instrumentation v1.11.1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.11.1) <br/><br/> Contains the [ADOT Collector for Lambda v0.17.0](https://github.com/aws-observability/aws-otel-collector/releases/tag/pkg%2Flambdacomponents%2Fv0.17.0) |
```
Comment (bryan-aguilar) [inline src/docs/getting-started/lambda/lambda-java.mdx:42]:
```suggestion
| ap-northeast-1<br/>ap-northeast-2<br/>ap-south-1<br/>ap-southeast-1<br/>ap-southeast-2<br/>ca-central-1<br/>eu-central-1<br/>eu-north-1<br/>eu-west-1<br/>eu-west-2<br/>eu-west-3<br/>sa-east-1<br/>us-east-1<br/>us-east-2<br/>us-west-1<br/>us-west-2 | arn:aws:lambda:\<region\>:901920570463:layer:aws-otel-java-wrapper-amd64-ver-1-11-1:1 | Contains [OpenTelemetry for Java v1.11.1](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.11.1) with [Java Instrumentation v1.11.1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.11.1) <br/><br/> Contains the [ADOT Collector for Lambda v0.17.0](https://github.com/aws-observability/aws-otel-collector/releases/tag/pkg%2Flambdacomponents%2Fv0.17.0) |
```

**Review by bryan-aguilar — COMMENTED** [review]
Are we standardizing the term `ADOT Collector for Lambda`? I am just a little confused if this is semantically correct.

**Comment by vasireddy99** [comment]
> Are we standardizing the term `ADOT Collector for Lambda`? I am just a little confused if this is semantically correct.

yeah, it was standardized since we are using the components from the ADOT collector.

File: src/docs/getting-started/lambda/lambda-dotnet.mdx [hunk 2]
```diff
@@ -65,11 +65,17 @@ This layer includes a reduced version of the [AWS Distro for OpenTelemetry Colle
 Note: Lambda layers are a regionalized resource, meaning that they can only be used in the Region in which they are published.
  Make sure to use the layer in the same region as your Lambda functions.
 
-Find the supported regions and ARN in the table below for the ARNs to consume.
+Find the supported regions and AMD64 layer ARN in the table below for the ARNs to consume.
```
Comment (NathanielRN) [inline src/docs/getting-started/lambda/lambda-dotnet.mdx:68]:
```suggestion
Find the supported regions and AMD64 (x86_64) layer ARN in the table below for the ARNs to consume.
```

File: src/docs/getting-started/lambda/lambda-go.mdx [hunk 3]
```diff
@@ -83,11 +83,17 @@ This layer includes a reduced version of the [AWS Distro for OpenTelemetry Colle
 Note: Lambda layers are a regionalized resource, meaning that they can only be used in the Region in which they are published.
  Make sure to use the layer in the same region as your Lambda functions.
 
-Find the supported regions and ARN in the table below for the ARNs to consume.
+Find the supported regions and AMD64 layer ARN in the table below for the ARNs to consume.
```
Comment (NathanielRN) [inline src/docs/getting-started/lambda/lambda-go.mdx:86]:
```suggestion
Find the supported regions and AMD64 (x86_64) layer ARN in the table below for the ARNs to consume.
```

File: src/docs/getting-started/lambda/lambda-java-auto-instr.mdx [hunk 4]
```diff
@@ -25,11 +25,17 @@ In this section, we consume the Lambda layer for use with Java Lambda Functions.
...
+
+Find the supported regions and ARM64 layer ARN in the table below for the ARNs to consume.
+
+|Supported   Regions  |Lambda layer ARN format  | Contents |
+|---------------------|-------------------------|----------|
+| ap-northeast-1<br/>ap-south-1<br/>ap-southeast-1<br/>ap-southeast-2<br/>eu-central-1<br/>eu-west-1<br/>eu-west-2<br/>us-east-1<br/>us-east-2<br/>us-west-2 | arn:aws:lambda:\<region\>:901920570463:layer:aws-otel-java-agent-amd64-ver-1-11-1:1 |  Contains [ADOT Java Auto-Instrumentation Agent v1.11.1](https://github.com/aws-observability/aws-otel-java-instrumentation/releases/tag/v1.11.1) <br/><br/> Contains the [ADOT Collector for Lambda v0.17.0](https://github.com/aws-observability/aws-otel-collector/releases/tag/pkg%2Flambdacomponents%2Fv0.17.0)|
```
Comment (NathanielRN) [inline src/docs/getting-started/lambda/lambda-java-auto-instr.mdx:38]:
```suggestion
| ap-northeast-1<br/>ap-south-1<br/>ap-southeast-1<br/>ap-southeast-2<br/>eu-central-1<br/>eu-west-1<br/>eu-west-2<br/>us-east-1<br/>us-east-2<br/>us-west-2 | arn:aws:lambda:\<region\>:901920570463:layer:aws-otel-java-agent-arm64-ver-1-11-1:1 |  Contains [ADOT Java Auto-Instrumentation Agent v1.11.1](https://github.com/aws-observability/aws-otel-java-instrumentation/releases/tag/v1.11.1) <br/><br/> Contains the [ADOT Collector for Lambda v0.17.0](https://github.com/aws-observability/aws-otel-collector/releases/tag/pkg%2Flambdacomponents%2Fv0.17.0)|
```

File: src/docs/getting-started/lambda/lambda-java-auto-instr.mdx [hunk 5]
```diff
@@ -25,11 +25,17 @@ In this section, we consume the Lambda layer for use with Java Lambda Functions.
 
 Note: Lambda layers are a regionalized resource, meaning that they can only be used in the region in which they are published. Make sure to use the layer in the same region as your Lambda functions.
 
-Find the supported regions and ARN in the table below for the ARNs to consume.
+Find the supported regions and AMD64 layer ARN in the table below for the ARNs to consume.
```
Comment (NathanielRN) [inline src/docs/getting-started/lambda/lambda-java-auto-instr.mdx:28]:
```suggestion
Find the supported regions and AMD64 (x86_64) layer ARN in the table below for the ARNs to consume.
```

File: src/docs/getting-started/lambda/lambda-java.mdx [hunk 6]
```diff
@@ -35,12 +35,17 @@ In this section, we consume the Lambda layer for use with Java Lambda Functions.
 Note: Lambda layers are a regionalized resource, meaning that they can only be used in the Region in which they are published.
  Make sure to use the layer in the same region as your Lambda functions.
 
-Find the supported regions and ARN in the table below for the ARNs to consume.
+Find the supported regions and AMD64 layer ARN in the table below for the ARNs to consume.
```
Comment (NathanielRN) [inline src/docs/getting-started/lambda/lambda-java.mdx:38]:
```suggestion
Find the supported regions and AMD64 (x86_64) layer ARN in the table below for the ARNs to consume.
```

File: src/docs/getting-started/lambda/lambda-js.mdx [hunk 7]
```diff
@@ -27,9 +27,17 @@ Note: Lambda layers are a regionalized resource, meaning that they can only be u
 
 Find the supported regions and ARN in the table below for the ARNs to consume.
 
-|Supported   Regions |Lambda layer ARN format| Contents |
+Find the supported regions and AMD64 layer ARN in the table below for the ARNs to consume.
```
Comment (NathanielRN) [inline src/docs/getting-started/lambda/lambda-js.mdx:30]:
```suggestion
Find the supported regions and AMD64 (x86_64) layer ARN in the table below for the ARNs to consume.
```

File: src/docs/getting-started/lambda/lambda-python.mdx [hunk 8]
```diff
@@ -28,11 +28,17 @@ In this section, we consume the Lambda layer for use with Python Lambda Function
 
 Note: Lambda layers are a regionalized resource, meaning that they can only be used in the Region in which they are published. Make sure to use the layer in the same region as your Lambda functions.
 
-Find the supported regions and ARN in the table below for the ARNs to consume.
+Find the supported regions and AMD64 layer ARN in the table below for the ARNs to consume.
```
Comment (NathanielRN) [inline src/docs/getting-started/lambda/lambda-python.mdx:31]:
```suggestion
Find the supported regions and AMD64 (x86_64) layer ARN in the table below for the ARNs to consume.
```

**Review by NathanielRN — CHANGES_REQUESTED** [review]
Looks great, only one critical comment about the Java ARM64 ARN.

**Review by alolita — APPROVED** [review]

**Pull request merged**